packages/mdc-select/foundation.ts

```diff
@@ -346,14 +373,36 @@ export class MDCSelectFoundation extends MDCFoundation<MDCSelectAdapter> {
     return this.adapter_.getSelectedTextAttr('aria-required') === 'true';
   }
 
-  setupMenu() {
+  init() {
+    const anchorEl = this.adapter_.getAnchorElement();
+    if (anchorEl) {
+      this.adapter_.setMenuAnchorElement(anchorEl);
+      this.adapter_.setMenuAnchorCorner(Corner.BOTTOM_START);
+    }
     this.adapter_.setMenuWrapFocus(false);
 
     const value = this.getValue();
     if (value) {
       this.setValue(value);
     }
   }
+
+  /**
+   * Unfocuses the select component.
+   */
+  private blur_() {
+    this.adapter_.removeClass(cssClasses.FOCUSED);
+    this.handleChange(false);
```

Comment:
We also need to remove calls to `handleChange` foundation method. 

`handleChange()` currently seem to have a didChange boolean flag that is used to trigger notify event, instead handleChange method should always trigger an event on change.

My understanding is that didChange was required to notch the outline on initial DOM sync which can be abstracted out into `init` or `layout` methods.

Sorry for going back & forth on this, but this refactor puts select component implementation in much better state and makes it much easier to wrap.

Reply:
Good catch! I moved that logic out into its own method `updateLabel`, so `handleChange` no longer needs any parameters.